### ISSUE TITLE
atags not loaded

### ISSUE THREAD

**Issue by Dan12** [jsandler18/raspi-kernel#2]
When I go in to debug or just print out the value of atags in kernel.c after completing step 5 I always get an address of 0 for the atags variable and mem_init doesn't work because the mem_size that it computes it 0.

This seems related:
https://seasonofcode.com/posts/qemu-arm-boot-tags.html

**Comment by jsandler18** [comment]
Yeah, the emulator is missing some key features. I included that it doesn't implement the system timer but I forgot about the atags. I'll have to include some.kind of standard warning in the tutorial about the limitations of the VM. Thanks for bringing this to my attention

**Comment by Dan12** [comment]
Since atags are just used for memory size, I think one workaround would be to modify ```get_mem_size``` in atag.c to return 128mb or some lower power of 2, which is what the makefile sets the memory size to in the emulator.

**Comment by jsandler18** [comment]
That's actually what I had originally but I did not wind up including it.
